spec/datetimeformat.html

```diff
@@ -383,7 +387,7 @@ <h1>FormatDateTimePattern ( _dateTimeFormat_, _patternParts_, _x_, _rangeFormatO
             1. If _p_ is *"hour"* and _dateTimeFormat_.[[HourCycle]] is *"h24"*, then
               1. If _v_ is 0, let _v_ be 24.
             1. If _f_ is *"numeric"*, then
-              1. Let _fv_ be FormatNumeric(_nf_, _v_).
+              1. Let _fv_ be FormatNumeric(_nf1_, _v_).
```

Comment:
We could probably just directly access the internal slot instead of using variables at this point. 
```suggestion
              1. Let _fv_ be FormatNumeric(_dateTimeFormat_.[[NumberFormat1]], _v_).
```